Loop/Views/InsulinModelSelection.swift

```diff
@@ -98,35 +94,35 @@ struct InsulinModelSelection: View, HorizontalSizeClassOverride {
                         )
                         .frame(height: 170)
 
-                        InsulinModelSettingsRow(
-                            title: Text("Rapid Acting - Adult", comment: "Title for rapid acting adult insulin model"),
-                            description: Text("This model assumes peak insulin activity at 75 minutes.", comment: "Description for rapid acting adult insulin model"),
+                        CheckmarkListItem(
+                            title: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).title),
+                            description: Text(InsulinModelSettings.exponentialPreset(.humalogNovologAdult).subtitle),
```

Comment:
Your LoopKit PR seems to extract the old titles/descriptions (which I failed to remove when redoing this screen); the ones included in InsulinModelSelection here are the updated versions per the design spec

Reply:
+1

Reply:
+1

Reply:
Will fix and update.